android/library/build.gradle

```diff
@@ -9,6 +10,7 @@ android {
 		targetSdkVersion rootProject.ext.targetSdkVersion
 		versionCode rootProject.ext.versionCode
 		versionName rootProject.ext.versionName
+		multiDexEnabled true
```

Comment:
argh, can you create a JIRA issue about it and assign it to Victor Galan?

Reply:
@nhpatt I don't have permissions, it is restricted to me

Reply:
@nhpatt @victorg1991 https://issues.liferay.com/browse/LSR-1130